FileExtensions.tex

```diff
@@ -0,0 +1,113 @@
+
+%&pdfLaTeX
+% !TEX encoding = UTF-8 Unicode
+\documentclass[a4paper]{article}
+\usepackage{ifxetex}
+\ifxetex
+\usepackage{fontspec}
+\setmainfont[Mapping=tex-text]{STIXGeneral}
+\else
+\usepackage[T1]{fontenc}
+\usepackage[latin1]{inputenc}
+\fi
+\usepackage{textcomp}
+
+\usepackage{graphicx}
+\usepackage{array}
+\usepackage{ulem}
+\usepackage{fixltx2e}
+\usepackage{amssymb}
+\usepackage{fancyhdr}
+\renewcommand{\headrulewidth}{0pt}
+\renewcommand{\footrulewidth}{0pt}
+
+\setlength{\parindent}{0cm}
+\setlength{\parskip}{0.18cm}
+\usepackage[hmargin=2cm,vmargin=2.5cm,bindingoffset=0.0cm]{geometry}
+\usepackage[pdfborder={0 0 0}]{hyperref}
+\begin{document}
+
+
+
+
+\input{FileExtensions.ver}
+\title{File Extensions (version 0.1)}
+\author{farjoun@broadinstitute.org}
+\date{\headdate}
+\maketitle
+
+\begin{quote}\small
+The master version of this document can be found at
+\url{https://github.com/samtools/hts-specs}.\\
+This printing is version~\commitdesc\ from that repository,
+last modified on the date shown above.
+\end{quote}
+
+\begin{center}
+\textit{license: Apache 2.0}
+\end{center}
+
+\section{\textbf{Overview}}
+This document's purpose is to collect all the file extensions that are supported/recomended for SAM-related files.
+
+
+The currently supported index formats are listed in table \ref{tab:index}.
+The table is organized in blocks, where the first line in each block describes the file that might be indexed, and the 
+remaining lines describe the possible index formats \& file extensions for said file. 
+The order of the index suffixes is the recommended order by which an implementation will search for an index file.
+
+\begin{table}
+\begin{center}
+\begin{tabular}{|c|c|c|c|}
+\hline
+\textbf{File Type} & \textbf{Algorithm} &\textbf{File extension}& \textbf{Specified in} \tabularnewline
+\hline
+SAM file &plain& {\tt .sam} & SAM \tabularnewline
+\hline
+CRAM file &cram& {\tt.cram} &CRAM\tabularnewline
+CRAM index &cram index&  {\tt .cram.crai} & CRAM+SAM  \\
```

Comment:
I don't understand the distinction of CRAM+SAM vs CRAM+BAM.  The .crai is defined in CRAM only, while .bai is CRAM spec with SAM/BAM spec for indexing.  Hence the first one should be "CRAM" only instead of "CRAM+SAM".